plotly/offline/offline.py

```diff
@@ -185,3 +165,12 @@ def plot():
     """ Configured to work with localhost Plotly graph viewer
     """
     raise NotImplementedError
+
+
+try:
+    _init_notebook_mode()
```

Comment:
@theengineear - note, this is a little bit scary because this gets run *every* time someone does:

`import plotly`

so we really need to make sure that no unexpected errors will be thrown here. Note that I'm catching the Ipython `ImportError` below, and the `PlotlyOfflineNotFound` error